optimism/src/lookup.rs

```diff
@@ -114,3 +114,27 @@ pub trait Lookups {
     /// Adds all lookups of Self to the environment
     fn lookups(&mut self);
 }
+
+/// A table of values that can be used for a lookup, along with the ID for the table.
```

Comment:
There's no ID in this struct... is there supposed to be?

Reply:
Oh that's an old comment